apps/explorer/src/ui/stories/ProgrammableTransactionCommands.stories.tsx

```diff
@@ -0,0 +1,153 @@
+            },
+        },
+    },
+};
+
+export const MakeMoveVec: StoryObj<
+    ICommandProps<(SuiArgument | SuiArgument[])[]>
+> = {
+    args: {
+        type: CommandType.MakeMoveVec,
+        data: [
+            [
+                {
+                    Result: 0,
+                },
+                {
+                    Result: 1,
+                },
+                {
+                    Result: 2,
+                },
+                {
+                    Result: 3,
+                },
+                {
+                    Result: 4,
+                },
+            ],
+        ],
+    },
+};
+
+export const MoveCall: StoryObj<ICommandProps<MoveCallSuiCommand>> = {
```

Comment:
IMO page-level stories provide a little less value, and are hard to maintain. I'd just remove these and write an e2e test if you want this value.